### ISSUE TITLE
hip fix 

### ISSUE THREAD

**Issue by lxy5513** [facebookresearch/VideoPose3D#51]
Hi, I get wild video 3D pose result, find the center hip point is fixed, when a person sit down, the hip points is same. like this: 

![test](https://user-images.githubusercontent.com/39720528/59990252-fad2c300-9674-11e9-9c36-787676789bd3.gif)

I think maybe is training data prepare, Could you please tell me what the function of inputs_3d[:, :, 0] = 0  
https://github.com/facebookresearch/VideoPose3D/blob/master/run.py#L312
why need set center_hip coordinate to (0,0,0), Is this aim to fix the hip point?  

Appreciate your apply Thank you in advance.

**Comment by dariopavllo** [comment]
That's how the model is trained and how evaluation is done. The hip joint is on the origin, and the predicted pose is relative to the origin.

You should train the trajectory model as well (we don't currently have pretrained models for that) if you want to recover the global position, but the accuracy may not be very good.

**Comment by lxy5513** [comment]
Thanks for your reply.  
but in https://dariopavllo.github.io/VideoPose3D/   
Why some demo videos hip joint isn't on the origin, It can move. I am confused about this.

**Comment by dariopavllo** [comment]
In that case, we simply added a constraint to force the height of the lowest joint to 0, so that the subject is always touching the ground.

For the same effect, you can replace https://github.com/facebookresearch/VideoPose3D/blob/1551a01dc254e51d4edbdbed7c7ff2ee4d9d5e2b/run.py#L729 with:
 `prediction[:, :, 2] -= np.min(prediction[:, :, 2], axis=1, keepdims=True)`

**Comment by lxy5513** [comment]
Appreciate you!
